Adding nrf_cc310_mbedcrypto version 0.8.1 (#65)
NordicPlayground/nrfxlib: crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

All notable changes to this project are documented in this file.

nrf_cc310_mbedcrypto - 0.8.1
****************************

> New experimental version of nrf_cc310_mbedcrypto with general bugfixes. 

Reviewer: ```suggestion
New experimental version of nrf_cc310_mbedcrypto with general bug fixes. 
```

> ..note::

Reviewer: ```suggestion
.. note::
```

  This version should be used for nRF9160 devices. Use of earlier versions may lead to 

>   undefined behanvior on some nRF9160 devices.

Reviewer 2: typo behanvior

Author: Fixed

Added
=====

Added new build of nRF cc310 mbed crypto library for nRF9160 and nRF52 architectures.

> ..note:

Reviewer: ```suggestion
.. note:
```
